recipes/autorift/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "autorift" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/leiyangleon/autoRIFT/archive/v{{ version }}.tar.gz
+  sha256: de5656fd5c9476635cc8d26ff0a323da2c8ed92ec2d92b529504f89bafde91fd
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win or py2k]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - opencv 3.4.*
+  run:
+    - python
+    - opencv 3.4.*
+    - libopencv 3.4.*
+    - scipy
+    - scikit-image
+    - gdal
+
+test:
+  imports:
+    - autoRIFT
+    - autoRIFT.autoRIFT
+    - autoRIFT.autoriftcore
+  commands:
+    - GeogridOptical.py -h
+
+about:
+  home: https://github.com/leiyangleon/autoRIFT
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: 'Fast pixel displacement estimation from remote sensing imagery'
+  doc_url: https://github.com/leiyangleon/autoRIFT
+  dev_url: https://github.com/leiyangleon/autoRIFT
+
+extra:
+  recipe-maintainers:
+    - leiyangleon
```

Comment:
@leiyangleon Would you like to be listed as a maintainer of this recipe?

Reply:
Yes, please add me as a maintainer of this recipe. Thanks!